### PR TITLE
KFSPTS-29789 fix jaggaer XML job PO mode to find vendors paid on one PO

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java [hunk 1]
```diff
@@ -245,7 +245,7 @@ private CuSqlChunk buildPurchaseOrderLimitSubQuery(Date processingDate) {
                 "FROM KFS.PUR_PO_T ",
                 "WHERE PO_LST_TRNS_DT > ",
                 CuSqlChunk.forParameter(Types.DATE, processingDate),
-                " GROUP BY VNDR_HDR_GNRTD_ID HAVING COUNT(1) > 1)");
+                " GROUP BY VNDR_HDR_GNRTD_ID HAVING COUNT(1) >= 1)");
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java:248]: changing this condition from `> 1` to `>= 1` might make the whole `HAVING` clause redundant, though I agree that writing it this way seems to make the business logic clearer. Please let me know if you'd rather leave it like this or if you think it should be changed.

**Review by cah292 — COMMENTED** [review]
I wanted to confirm one quick item before merging.

File: src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java (hunk 1 above)
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java:248]: I think the improved clarity of leaving the code like this is worth it, would you agree?

Comment (cah292) [reply src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java:248]: Leaving it like this is fine with me. Thanks for confirming!

**Review by cah292 — APPROVED** [review]
Changes look good and unit tests pass. Merging.

**Pull request merged**